### PR TITLE
fix: add a pa status activity for pipelines which have timed out

### PR DESCRIPTION
**Pull request by ankitm123**
Signed-off-by: ankitm123 <ankitmohapatra123@gmail.com>

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -1,30 +1,10 @@
 approvers:
 - rawlingsj
-- jstrachan
```
Comment (rawlingsj) [inline OWNERS:3]: Let's leave JS

**Comment by rawlingsj** [comment]
LGTM with the one comment addressed

File: OWNERS [hunk 2]
```diff
@@ -1,30 +1,10 @@
...
-- rajdavies
-- ccojocar
-- garethjevans
```
Comment (rawlingsj) [inline OWNERS:6]: And GE too

File: OWNERS (hunk 1 above)
Comment (ankitm123) [reply OWNERS:3]: sounds good

**Comment by ankitm123** [comment]
@rawlingsj Added them back :)

**Pull request merged**